README.md

```diff
@@ -14,9 +15,49 @@ Just add this to your `Cargo.toml`:
 hibitset = "0.6"
 ```
 
+## Using the `serde` feature
+
+There is an optional feature to use `serde` for serialization/deserializtion. To enable this feature add the following to your `Cargo.toml`:
+
+```
+[dependencies]
+hibitset = { version = "0.6.3", features = ["serde"]}
+serde_derive = "1.0.111"
+serde = "1.0.111"
+bincode = "1.2.1"
+```
+
+Using `bincode` here is an example of how you can serialize/deserialize a `BitSet`:
+
+```
+#[macro_use]
+extern crate serde_derive;
+
+use hibitset::{BitSet, BitSetAnd, BitSetLike, BitSetNot};
```

Comment:
There are some imports here that are unnecessary for  the example.

Reply:
I removed the unnecessary imports in the example.